lib/pharos/resources/openebs/30-default-storage-pool.yml.erb

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: openebs.io/v1alpha1
+kind: StoragePool
+metadata:
+  name: default
+  type: hostdir
+spec:
+  path: <%= default_storage_pool_path %>
+  nodes: [ "all" ]
+  #maxVolSize: 10G #Max Volume Size
+  #maxAllocatedSize: 100G #Max Total Space allocated from this directory/mount
+---
```

Comment:
Trailing document separator

Reply:
removed, leftover from initial try to put many resources into same yaml file.